search/views.py

```diff
@@ -87,3 +86,9 @@ def subsq(a, b, m, n):
         return subsq(a, b, m - 1, n - 1)
     # If last characters are not matching
     return subsq(a, b, m, n - 1)
+
+def display(request):
+    if request.method == 'POST':
+        display = request.POST.get('path')
+    r = requests.get(display)
+    return render(request, 'cosmos/data.html',{'code':r.text})
```

Comment:
It looks like you have not included `data.html` in this pull request. Kindly include it.